### PR TITLE
Fix PHP 7.2 tests on master branch

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | not yet
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

**Review by Simperfit — APPROVED** [review]

**Review by sroze — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -20,10 +20,7 @@ env:
 
 matrix:
     include:
-        - php: 7.1.3
-        - php: 7.1
-          env: deps=high
-        - php: 7.1
```
Comment (stof) [inline .travis.yml]: this is meant to be reverted, right ?

**Comment by stof** [comment]
Looks weird to have @Simperfit and @sroze approving the PR at a time it is totally not mergeable...

File: .travis.yml (hunk 1 above)
Comment (nicolas-grekas) [reply .travis.yml]: yes, just for testing now

**Comment by nicolas-grekas** [comment]
On 7.2:
- HttpFoundations's NativeSessionStorageTest fails with "ini_set(): A session is active. You cannot change the session module's ini settings at this time"
- Process hangs completely.
If someone wants to investigate, help welcomed :)

**Comment by Simperfit** [comment]
@nicolas-grekas Did you merge the one we did ? it's the same thing

**Comment by nicolas-grekas** [comment]
Yes I did, so this is some leftover.

**Comment by Simperfit** [comment]
Both the ini's one should not be hard to fix (I think it's the same error as the other PR).

About the Process components that hang, I could help you debugging if you want to.

@stof I approuved this PR because I though (maybe, too fast) that this will be working directly since we already worked on it. Sorry.

**Comment by stof** [comment]
@Simperfit removing testing on PHP 7.1 cannot be approved as is anyway.

**Comment by sroze** [comment]
@nicolas-grekas Interestingly I see it the other way around:
1. The Process tests failing (fix proposed in #24516)
2. HttpFoundation hanging. Will dig into this 🤔

**Comment by nicolas-grekas** [comment]
If it hangs locally with HttpFoundation, it may be related to an obsolete phpunit bridge. Try removing the `.phpunit` folder locally.

**Comment by sroze** [comment]
Doesn't seam like it, I've `dev-master` with your commit c754fc1 in it (tried the `rm` option as well)

**Comment by sroze** [comment]
Looks like all the tests tagged `@runInSeparateProcess` are blocking actually.

**Comment by nicolas-grekas** [comment]
`composer update`

**Pull request closed without merging**